src/Example2/Domain/Repository.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Example2\Domain;
+
+interface Entity{}
```

Comment:
All looks great, just one little thing. Entity interface should be on it's own separate file (otherwise it won't be able to load - composer autoloader says: one interface / class per file) I will do it

Reply:
P.D: it actually loads :D but will update it given that it's better practise in php world

Reply:
![image](https://user-images.githubusercontent.com/8798694/104843152-628cd680-58c9-11eb-99ba-4b3fb485df0b.png)